vagga.yaml

```diff
@@ -0,0 +1,27 @@
+minimum-vagga: v0.7.0
+containers:
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.8'
+      - 'gunicorn >=19.6.0'
+      - 'whitenoise >=3.2.3'
+      - 'django-environ >=0.4.1'
+    - !Sh pip freeze > requirements.txt
```

Comment:
You probably want to write `-r base-requirements.txt` to the file first and then append the `freeze` list using `>>` operator